app/src/main/java/com/framgia/fsalon/screen/scheduler/SchedulerViewModel.java

```diff
@@ -225,14 +256,71 @@ public void hideLoadMore() {
     }
 
     @Override
-    public void onClickTopSheet(View topSheet) {
-        TopSheetBehavior.from(topSheet).setState(TopSheetBehavior.STATE_COLLAPSED);
+    public void onFilterClick(DrawerLayout layout) {
+        if (layout.isDrawerOpen(Gravity.LEFT)) {
+            layout.closeDrawer(Gravity.LEFT);
+            return;
+        }
+        layout.openDrawer(Gravity.LEFT);
```

Comment:
WARNING: [RtlHardcoded] Use "`Gravity.START`" instead of "`Gravity.LEFT`" to ensure correct behavior in right-to-left locales
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)